desktop/cmp/dash/canvas/impl/DashCanvasContextMenu.js

```diff
@@ -75,10 +76,14 @@ function createAddMenuItems({dashCanvasModel, clickPosition}) {
                 item = {
                     text: title,
                     icon: icon,
-                    actionFn: () => dashCanvasModel.addView(
-                        id,
-                        {layout: addPosition}
-                    )
+                    actionFn: () => {
+                        const viewModel = dashCanvasModel.addView(
+                            id,
+                            {layout: addPosition}
+                        );
+                        // Delay allows view to render before scrolling into view
+                        delay(() => viewModel.ref.current.scrollIntoView({behavior: 'smooth'}), 1);
```

Comment:
perhaps add a method on DashCanvasModel of the form 'ensureVisible' can imagine this is generally useful.  also prefer `wait().then()` for this construct. A number of advantages: 
+ Putting the number first more readable and having the function last allows easier formatting. 
+ Also can easily  use promise changes, await and exception handling